lib/cli/index.js

```diff
@@ -7,6 +7,10 @@ var logger        = require('heimdalljs-logger')('ember-cli:cli/index');
 var merge         = require('ember-cli-lodash-subset').merge;
 var path          = require('path');
 
+// work around misbehaving libraries, so we can correctly cleanup before
+// actually exiting.
+require('capture-exit').captureExit();
```

Comment:
Didn't think about other picking off `process.exit` in advance. Manual review says that isn't a concern here, but it could accidentally become a problem in the future if not hoisted. Seems good to be addressed.